tests/cases/text/textinput_functions.slint

```diff
@@ -6,21 +6,24 @@ export component TestCase {
     height: 100phx;
 
     ti := TextInput {}
-    
+
     in-out property<string> test_text <=> ti.text;
 
-    public function do_select() {
+    callback do_select();
+    do_select => {
```

Comment:
I'm curious, any particular reason for this change?

Reply:
Oh, I see now. Whoops.

Reply:
As i wrote in the commit message, the node bindings currently don't allow to invoke functions :-(
(we really should re-vamp our node API, because now, i can't even run them locally anymore because my version of python is too recent to build the bindings :-()